### PR TITLE
Release/1.0.2

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Broadcasting changes.

Requirement:
1. To use this widget in Global Visibility Dashboard
2. This widget should listen to an on-click event of "Record Summary Card" if configured with the same event name in advanced settings

Implementation :
1. Added an event listener for the widget. Which would listen to the events broadcasted by the "Record Summary Card".

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -2,81 +2,129 @@
...
       else {
         populateCustomData();
       }
     }
 
+    //if this event is supposed to listen to eny of the broadcasted events
+    if (_config.broadcastEvent) {
+      //example widget:globalVisibilityEvent from Record Summary card
+      $rootScope.$on("widget:"+_config.eventName, function (event, data) {
+        var element = document.getElementById("funnelChartParentDiv" + _config.wid);
```
Comment (cs-bk) [inline widget/view.controller.js]: keep this in a separate funciton

File: widget/edit.html [hunk 2]
```diff
@@ -153,13 +164,63 @@ <h6 style="margin-top: 0px;">Layer {{$index +1}}  <div class="display-inline-blo
...
+
+                <uib-accordion>
+                    <div uib-accordion-group class="queue-card-wrapper panel-default padding-top-lg">
+                        <uib-accordion-heading>
+                            <div class="queue-card-header padding-left-md workspace-connector-metadata-container"
+                                ng-click="toggleArrow()" data-ng-class="{'queue-text-active-color': toggle">
+                                <div class="margin-right-lg pull-left"><span class="queue-card-actions">&nbsp;<i
+                                            class="fa {{ toggle ? 'fa-chevron-down' : 'fa-chevron-right'}}"></i></span>
+                                </div>
+                                <div class="pull-left queue-card-header-text margin-right-3">Advanced Settings
```
Comment (cs-bk) [inline widget/edit.html:186]: @cs-yash-patodkar - wrap advanced settings

File: widget/view.controller.js [hunk 3]
```diff
@@ -151,7 +199,6 @@
         margin = margin + 15;
         width = width - 30;
       }
-
     }
   }
 })();
```
Comment (cs-bk) [inline widget/view.controller.js:216]: @cs-yash-patodkar - view failing for 'count of records across module

Screen Shot 2023-07-27 at 11 45 58 AM
'

**Review by cs-bk — APPROVED** [review]
LGTM

**Review by cs-suyog-jinturkar — APPROVED** [review]
BK already approved this PR, hence marking it as approved here as well.

**Pull request merged**